src/app/components/Enable/LiquidEnable.tsx

```diff
@@ -0,0 +1,122 @@
+import { CheckIcon } from "@bitcoin-design/bitcoin-icons-react/filled";
+import ConfirmOrCancel from "@components/ConfirmOrCancel";
+import Container from "@components/Container";
+import PublisherCard from "@components/PublisherCard";
+import { useCallback, useEffect, useRef, useState } from "react";
+import { useTranslation } from "react-i18next";
+import ScreenHeader from "~/app/components/ScreenHeader";
+import toast from "~/app/components/Toast";
+import { USER_REJECTED_ERROR } from "~/common/constants";
+import msg from "~/common/lib/msg";
+import type { OriginData } from "~/types";
+
+type Props = {
+  origin: OriginData;
+};
+function LiquidEnableComponent(props: Props) {
+  const hasFetchedData = useRef(false);
+  const [loading, setLoading] = useState(false);
+  const { t } = useTranslation("translation", {
+    keyPrefix: "enable",
+  });
+  const { t: tCommon } = useTranslation("common");
+
+  const enable = useCallback(() => {
+    try {
+      setLoading(true);
+      msg.reply({
+        enabled: true,
+        remember: true,
+      });
+    } catch (e) {
+      console.error(e);
+      if (e instanceof Error) toast.error(`${tCommon("error")}: ${e.message}`);
+    } finally {
+      setLoading(false);
+    }
+  }, [tCommon]);
+
+  function reject(event: React.MouseEvent<HTMLAnchorElement>) {
+    event.preventDefault();
+    msg.error(USER_REJECTED_ERROR);
+  }
+
+  async function block(event: React.MouseEvent<HTMLAnchorElement>) {
+    event.preventDefault();
+    await msg.request("addBlocklist", {
+      domain: props.origin.domain,
+      host: props.origin.host,
+    });
+    alert(t("block_added", { host: props.origin.host }));
+    msg.error(USER_REJECTED_ERROR);
+  }
+
+  useEffect(() => {
+    async function getAllowance() {
+      try {
+        const allowance = await msg.request("getAllowance", {
+          domain: props.origin.domain,
+          host: props.origin.host,
+        });
+        if (allowance && allowance.enabled) {
+          enable();
+        }
+      } catch (e) {
+        if (e instanceof Error) console.error(e.message);
```

Comment:
let's remove this. 
I think we already check this in the action and the prompt should not even open if there is an allowance. 

if it still opens, then we can show it I think. 

so we don't need this code.